docs/getting-started/introduction.rst

```diff
@@ -39,14 +39,15 @@ What do I need?
 ===============
 
 .. sidebar:: Version Requirements
-    :subtitle: Celery version 5.1 runs on
+    :subtitle: Celery version 5.2 runs on
 
-    - Python ❨3.6, 3.7, 3.8❩
+    - Python ❨3.7, 3.8, 3.9❩
```

Comment:
Py3.10 is also supported

Reply:
@auvipy Please use suggestions for changes this simple.